src/docs/product/relay/best-practices.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Self-hosting - Best Practices
+sidebar_order: 5
+---
+
+This page contains some advices and best practices when self-hosting external Relays, i.e. Relays that run on your hardware and forward events to `sentry.io`.
+
+## General Considerations
+
+* While Relay can be built from source, the recommended way to run Relay is via the officially provided Docker image (`getsentry/relay`), which can be found on [DockerHub]. The images are tagged with their Git revision identifier.
```

Comment:
```suggestion
* We recommend running Relay using the officially provided Docker image (`getsentry/relay`) found on [DockerHub] and tagged with its Git revision identifier, rather than building from source.
```